### PR TITLE
Fix invalid date selection

### PR DESCRIPTION
**Pull request by ruwiniwj**
## Purpose
Fix issue where invalid dates are selected in custom range selector

### CODE REVIEW

File: components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx [hunk 1]
```diff
@@ -101,10 +102,24 @@ export default class CustomTimeRangeSelector extends React.Component {
...
+        } else {
+            this.setState({ invalidDateRange: false });
+        }
     }
 
     handleEndTimeChange(date) {
         this.endTime = date;
+        const start = Moment(this.startTime);
+        const end = Moment(this.endTime);
+        if(end.diff(start) < 0) {
```
Comment (TanyaM) [inline components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx]: formatting issues

File: components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx [hunk 2]
```diff
@@ -101,10 +102,24 @@ export default class CustomTimeRangeSelector extends React.Component {
 
     handleStartTimeChange(date) {
         this.startTime = date;
+        const start = Moment(this.startTime);
+        const end = Moment(this.endTime);
+        if(end.diff(start) < 0) {
```
Comment (TanyaM) [inline components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx]: formatting issues

File: components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx (hunk 1 above)
Comment (ruwiniwj) [reply components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx]: This code is refactored with PR https://github.com/wso2/analytics-solutions/pull/270

File: components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx (hunk 2 above)
Comment (ruwiniwj) [reply components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx]: This code is refactored with PR https://github.com/wso2/analytics-solutions/pull/270

**Pull request merged**